piker/_daemon.py

```diff
@@ -0,0 +1,79 @@
+"""
+pikerd daemon lifecylcle & rpc
+"""
+from typing import Optional
+from contextlib import asynccontextmanager
+
+import tractor
+
+from .log import get_logger, get_console_log
+from .brokers import get_brokermod
+
+
+log = get_logger(__name__)
+
+root_nursery: Optional[tractor._trionics.ActorNursery] = None
+root_dname = 'pikerd'
+root_modules = [
+    __name__,
+    'piker._ems'
+]
+
+@asynccontextmanager
+async def maybe_spawn_pikerd(
```

Comment:
Hmm looking at this further I wonder if we should choose a better name like `maybe_open_pikerd`?

Technically this doesn't actually spawn anything, just might open a nursery.